datastore/google/cloud/datastore/transaction.py

```diff
@@ -186,7 +186,7 @@ def begin(self):
         try:
             self._id = self._client._connection.begin_transaction(
                 self.project)
-        except:
+        except:  # noqa: E722 do not use bare except, specify exception instead
```

Comment:
Yeah, but that text is searchable.  e.g. if you copy and paste that into google you'll get more information and the source of the definition.